src/actions/requestsApi.js

```diff
@@ -4381,3 +4381,9 @@ export const voidPaymentRequest = {
 		buildUrl(["carts", scopeId, customerId, cartName, "payments", paymentId, "void"]),
 	verb: "POST",
 };
+
+export const createProfileDefinitonRequest = {
```

Comment:
this file we do not change manually but with script and the request is here with name:
createEntityTypeRequest